schemas/com.snowplowanalytics.mobile/application/jsonschema/1-0-0

```diff
@@ -0,0 +1,25 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for an application context",
+	"self": {
+		"vendor": "com.snowplowanalytics.mobile",
+		"name": "application",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+	"type": "object",
+	"properties": {
+		"version": {
+			"type": "string",
+			"description": "Version of the application e.g 1.1.0",
+			"maxLength": 64
+		},
+		"build": {
```

Comment:
I'm sorry, I'm not really aware of context (pun!) of this task, but I don't think that any application has a `build`, e.g. I cannot think of any build ids in my CLI nor Server apps, so I assume this is optional. Also description of `build` is not very useful.

Reply:
Sorry, I actually just noticed `mobile` in vendor, which both answers some of my questions and slightly confuses me. I think something helping to uniquely identify an application is very useful across all platforms, not just mobile. Beside of current tasks, is there any reasons to restrict it to be mobile-only?

Reply:
@chuwy apologies I probably should've given more context in my email earlier.

A big focus for the solutions engineering team is the Mobile Foundation (part of the Services OKR's). The whole idea behind Mobile Foundation is to decrease the time to value for our iOS / Android SDK users. The approach as a result is user-led. One decision that we made was to start using a separate vendor for any schema releases as it's a better user experience.

Fully agree that tracking app versions applies not just to Mobile apps but the "Mobile Foundation" release of the application context will be used just for our users of the iOS / Android trackers. @mhadam has already worked on functionality for the iOS/Android trackers to start tracking this automatically which is why we've written a dedicated schema.